src/recipe/test/e2e/findAllRecipes.controller.e2e-spec.ts

```diff
@@ -20,7 +21,7 @@ describe('Recipe Controller - Find All Recipes', () => {
 
   beforeAll(async () => {
     const moduleFixture: TestingModule = await Test.createTestingModule({
-      imports: [RecipeModule, AuthModule],
+      imports: [RecipeModule, AuthModule, AppModule],
```

Comment:
I assume that AppModule imports RecipeModule, AuthModule
Do you need to import Recipe and Auth modules separately?